packages/wdio-browserstack-service/package.json

```diff
@@ -30,9 +30,6 @@
     "got": "^11.0.2",
     "webdriverio": "7.20.3"
   },
-  "peerDependencies": {
-    "@wdio/cli": "^7.0.0"
```

Comment:
How about we say `"@wdio/cli": "^5.0.0"`?

Reply:
I am not sure if that would also include multiple version but basically ensure that we keep a restriction to v5 and above

Reply:
Ok, so since we are supporting for v5 to 7, this make more sense:
`"peerDependencies": {
    "@wdio/cli": "5 - 7"
  },`

Reply:
Ok, so since we are supporting for v5 to 7, does this make more sense:
`"peerDependencies": {
    "@wdio/cli": "5 - 7"
  },` 
  or `"peerDependencies": {
    "@wdio/cli": "^5.0.0 || ^6.0.0 || ^7.0.0"
  },`

Reply:
Let's do `"peerDependencies": { "@wdio/cli": "^5.0.0 || ^6.0.0 || ^7.0.0" },`